src/core/geometry/qgslinestring.cpp

```diff
@@ -1936,12 +1936,16 @@ QgsPoint QgsLineString::centroid() const
 
 void QgsLineString::sumUpArea( double &sum ) const
 {
+  // Please note that the area calculated in this method will be negative if the line points are defined in clockwise order.
```

Comment:
These comments would belong better as additions to the docstrings in the header file, so that they get included in the API docs and PyQGIS references.

Reply:
OK I have now added comments to the header file instead.